### DISCUSSION TITLE
Possible to styles mentions inserted in to editor?

### DISCUSSION THREAD

**Discussion by just-Bri** [afconsult/quill-mention#290] [Q&A] [answered]
Hello,

Picked up quill, react-quill, and quill-mention this morning to bang out a quick POC to replace our current text input and mention method which has been a giant pain to work with.  

So far I've got the basics in place, editor is working, mention list pops up with styles and is clickable/enter'able, it's being placed in the editor as desired.

What I'm having trouble finding is a way to style the mention in the editor:  
![image](https://user-images.githubusercontent.com/4626724/127750473-a514e702-b33b-4ff8-9869-7030277af08b.png)

Is there any way to style the mention when it is inserted in to the editor area? Our mention solution was a real hack to get working but after doing @Bridger and picking "Bridger Hammond" from the options this is what is added to the editor:  
![image](https://user-images.githubusercontent.com/4626724/127750573-dc1bba23-5f1b-4d58-b71d-d014cc1d30b4.png)

Is this currently posible?

**Comment by just-Bri** [comment]
Leaving this comment for others who are trying to do this:

I decided to just copy the "snow" theme .css file in to the folder of my component and directly target 
```
.ql-editor p > span > span {
  background-color: blue;
}
```
Which gives me this:  
![image](https://user-images.githubusercontent.com/4626724/127750866-cbc33857-bf64-434a-8421-e5a173721445.png)

Obviously not the final product, but servicable.

**Comment by zehawki** [comment, marked as answer]
Nothing to it. The inserted mention uses a class name "mention". Just CSS style it to whatever you want.
